### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by musakazem**
Added social media links(Facebook)  to the following speakers:
Michael Cheng
Alexandre Lision
Anivar Aravind

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1663,6 +1665,7 @@ <h1>Speakers</h1>
 										</div>
 										<div class="social-links vertical-align">
 											<a href="http://anivar.net"><i class="icon fa fa-external-link"></i></a>
+											<a href="https://www.facebook.com/an1var/?hc_ref=ARTVpb1ckvX2lz1BR2n25Adv_rcR9NCUy4SAp7otRXabqihX8JFYa4vUlMKgfhCUm0s&fref=nf"><i class="icon social_facebook"></i></a>
```
Comment (anshumanv) [inline index.html:1668]: Replace by this - https://www.facebook.com/an1var

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments.

**Comment by anshumanv** [comment]
Please reference the issue that you're solving and update the PR title to what this PR actually does, same with the commit.

**Review by akshatnitd — COMMENTED** [review]
@musakazem Please make the requested changes so that we can merge it. Also, you need to resolve the conflicts!

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@musakazem  Please resolve conflicts and update your PR.

**Comment by ParthS007** [comment]
closing due to inactivity 👍

**Pull request closed without merging**